packages/manager/apps/web/client/app/app.js

```diff
@@ -59,69 +58,72 @@ Environment.setRegion(__WEBPACK_REGION__);
 Environment.setVersion(__VERSION__);
 
 angular
-  .module('App', [
-    ovhManagerCore,
-    ngPaginationFront,
-    'ngOvhUtils',
-    'ui.bootstrap',
-    'ngAria',
-    'ngRoute',
-    'ngResource',
-    'ngSanitize',
-    'controllers',
-    'services',
-    'filters',
-    'directives',
-    ngQAllSettled,
-    'ngMessages',
-    'ngFlash',
-    'vs-repeat',
-    'xeditable',
-    ngAtInternet,
-    ngAtInternetUiRouterPlugin,
-    ngOvhApiWrappers,
-    ngOvhContracts,
-    // ngOvhChatbot,
-    ngOvhContracts,
-    ngOvhExportCsv,
-    ngOvhHttp,
-    ngOvhSsoAuth,
-    ngOvhSsoAuthModalPlugin,
-    ngOvhSwimmingPoll,
-    ngOvhProxyRequest,
-    ngOvhUserPref,
-    ngOvhWebUniverseComponents,
-    ngTranslateAsyncLoader,
-    ngUiRouterLayout,
-    ngUiRouterLineProgress,
-    ovhManagerServerSidebar,
-    uiRouter,
-    'pascalprecht.translate',
-    ngTailLogs,
-    'ovh-api-services',
-    ovhManagerMfaEnrollment,
-    ovhManagerBanner,
-    ovhManagerNavbar,
-    'moment-picker',
-    'oui',
-    'Module.exchange',
-    emailpro,
-    exchange,
-    office,
-    sharepoint,
-    'Module.emailpro',
-    domain,
-    domainDnsZone,
-    emailDomainOrder,
-    errorPage,
-    hosting,
-    orderCatalogPrice,
-    privateDatabase,
-    zone,
-    emailDomainUpgradeModule,
-    hostingEmail,
-    hostingEmailActivateModule,
-  ])
+  .module(
+    'App',
+    [
+      ovhManagerCore,
+      ngPaginationFront,
+      'ngOvhUtils',
+      'ui.bootstrap',
+      'ngAria',
+      'ngRoute',
+      'ngResource',
+      'ngSanitize',
+      'controllers',
+      'services',
+      'filters',
+      'directives',
+      ngQAllSettled,
+      'ngMessages',
+      'ngFlash',
+      'vs-repeat',
+      'xeditable',
+      ngAtInternet,
+      ngAtInternetUiRouterPlugin,
+      ngOvhApiWrappers,
+      ngOvhContracts,
+      ngOvhContracts,
+      ngOvhExportCsv,
+      ngOvhHttp,
+      ngOvhSsoAuth,
+      ngOvhSsoAuthModalPlugin,
+      ngOvhSwimmingPoll,
+      ngOvhProxyRequest,
+      ngOvhUserPref,
+      ngOvhWebUniverseComponents,
+      ngTranslateAsyncLoader,
+      ngUiRouterLayout,
+      ngUiRouterLineProgress,
+      ovhManagerServerSidebar,
+      uiRouter,
+      'pascalprecht.translate',
+      ngTailLogs,
+      'ovh-api-services',
+      ovhManagerMfaEnrollment,
+      ovhManagerBanner,
+      ovhManagerNavbar,
+      'moment-picker',
+      'oui',
+      'Module.exchange',
+      emailpro,
+      exchange,
+      office,
+      sharepoint,
+      'Module.emailpro',
+      domain,
+      domainDnsZone,
+      emailDomainOrder,
+      errorPage,
+      hosting,
+      orderCatalogPrice,
+      privateDatabase,
+      zone,
+      emailDomainUpgradeModule,
+      hostingEmail,
+      hostingEmailActivateModule,
+      __NG_APP_INJECTIONS__,
+    ].filter((value) => value !== null), // __NG_APP_INJECTIONS__ can be null
```

Comment:
Other application is using `_.isString`.

Could you please confirm that if `ngAppInjections` is undefined, it's equals to `null` and not `'null'` (see: webpack.config.js)?

Reply:
yes I will update it ;)

Reply:
* If there is no injection  `__NG_APP_INJECTIONS__` will be replaced by `null` .
* If there is `__NG_APP_INJECTIONS__` it will be replaced by `'moduleA'` or `'moduleA', 'moduleB'`